src/components/Personalization/constants/personalizationConstants.js

```diff
@@ -17,3 +17,4 @@ export const JSON_CONTENT_ITEM =
   "https://ns.adobe.com/personalization/json-content-item";
 export const REDIRECT_ITEM =
   "https://ns.adobe.com/personalization/redirect-item";
```

Comment:
A filename like `personalizationConstants.js` suggests that the focus of this file might not be narrow enough, especially since it's located under `Personalization/constants`. Rather than a bucket of all potential personalization constants, would it make more sense to have `schema.js` and `scope.js` files instead? Another thing to note is that `personalizationConstants.js` is plural whereas our convention is that the file should be singular (https://github.com/adobe/alloy/wiki/Conventions#enumerations).

Reply:
thank you, will extract those in separate files